addons/membership/models/membership.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models, fields, _
+import odoo.addons.decimal_precision as dp
```

Comment:
Order the imports according to [guidelines](https://www.odoo.com/documentation/9.0/reference/guidelines.html#imports).